src/lib/fetch-document-list.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-console */
+const { info } = require('../lib/log');
+const fs = require('../lib/sync-fs');
+const DOC_TYPES = ['district_hospital', 'health_center', 'clinic', 'person', 'user', 'user-settings', 'contact'];
```

Comment:
Minor readability request on this: Please rename this to indicate the usage of it. Yes, they are "doc types" but what is this list for?  You can bake that into the name so others can easily see what the list is all about, and if they need to change it, the name can hint as to if their change "feels right" or not.